packages/teams-js/test/internal/utils.spec.ts

```diff
@@ -164,4 +171,98 @@ describe('utils', () => {
       expect(createTeamsAppLink(params)).toBe(expected);
     });
   });
+
+  describe('base64ToBlob', () => {
+    it('should convert base64 string to Blob for image/png MIME type', async () => {
+      const base64Data = 'SGVsbG8=';
+
+      const data: ClipboardParams = {
+        mimeType: SupportedMimeType.ImagePNG,
+        content: base64Data,
+      };
+
+      const result = await base64ToBlob(data);
+
+      expect(result).toBeInstanceOf(Blob);
+      expect(result.type).toBe(SupportedMimeType.ImagePNG);
+    });
+
+    it('should convert base64 string to Blob for image/jpeg MIME type', async () => {
+      const base64Data = 'SGVsbG8=';
+
+      const data: ClipboardParams = {
+        mimeType: SupportedMimeType.ImageJPEG,
+        content: base64Data,
+      };
+
+      const result = await base64ToBlob(data);
+
+      expect(result).toBeInstanceOf(Blob);
+      expect(result.type).toBe(SupportedMimeType.ImageJPEG);
+    });
+
+    it('should convert base64 string to Blob for non-image MIME type', async () => {
+      const base64Data = 'SGVsbG8=';
+      const data: ClipboardParams = {
+        mimeType: SupportedMimeType.TextPlain,
+        content: base64Data,
+      };
+
+      const result = await base64ToBlob(data);
+
+      expect(result).toBeInstanceOf(Blob);
+      expect(result.type).toBe(SupportedMimeType.TextPlain);
+    });
+
+    it('should convert base64 string to Blob for non-image MIME type', async () => {
+      const base64Data = 'PHA+SGVsbG8sIHdvcmxkITwvcD4=';
+      const data: ClipboardParams = {
+        mimeType: SupportedMimeType.TextHtml,
+        content: base64Data,
+      };
+
+      const result = await base64ToBlob(data);
+
+      expect(result).toBeInstanceOf(Blob);
+      expect(result.type).toBe(SupportedMimeType.TextHtml);
+    });
+  });
+
+  describe('getBase64StringFromBlob', () => {
```

Comment:
similar comment here: would be good to check error cases for this function (null blob, promise is rejected, etc.)

Reply:
Updated in the latest commit